packages/contracts-bedrock/src/dispute/DisputeGameFactory.sol

```diff
@@ -132,4 +138,10 @@ contract DisputeGameFactory is OwnableUpgradeable, IDisputeGameFactory, ISemver
         gameImpls[_gameType] = _impl;
         emit ImplementationSet(address(_impl), _gameType);
     }
+
+    /// @inheritdoc IDisputeGameFactory
+    function setInitBond(GameType _gameType, uint256 _initBond) external onlyOwner {
```

Comment:
Perhaps this can be handled by `setImplementation`? Any reason we wanna set implementation and the initial bond separately?

Reply:
Think that this should be fine - coupling with `setImplementation` adds extra side effects to that function, and odds are the initial bonds won't always need to be upgraded. Can always change later if we feel the need :)